packages/services/storage/package.json

```diff
@@ -24,13 +24,15 @@
     "slonik": "30.4.4",
     "slonik-interceptor-query-logging": "1.4.7",
     "slonik-utilities": "1.9.4",
+    "ssri": "10.0.4",
```

Comment:
Do we really need this additional dependency? Couldn't we just us the Node.js `crypto` module similar to this code? https://github.com/kamilkisiela/graphql-hive/blob/fc441974d3a08d31e65fb63fc1aff8a8011a85f8/packages/libraries/client/src/apollo.ts#L54

Reply:
yeah, no prob with that :)